crates/cast/tests/cli/main.rs

```diff
@@ -3382,8 +3382,12 @@ contract WETH9 {
 
 // <https://github.com/foundry-rs/foundry/issues/10553>
 // <https://basescan.org/tx/0x17b2de59ebd7dfd2452a3638a16737b6b65ae816c1c5571631dc0d80b63c41de>
-casttest!(osaka_can_run_p256_precompile, |_prj, cmd| {
-    cmd.args([
+// Note(zk): Ignored - this test is also failing in upstream
```

Comment:
I managed to get this test to pass (at least I think I did) here 
https://github.com/matter-labs/foundry-zksync/pull/1190/commits/5416d563a4509fc4973350fe7737c14fe02b16ee#diff-492a009588464cc6024300f039dbe3a5e66b3ea0ac4db8f57ed779c4388693d9R3402
and then applying this: (basically reverting the `from`, `to`, etc.) 
https://github.com/matter-labs/foundry-zksync/pull/1190/commits/61debd684322d19b8f89b34acd7067e3d541c1e8

You can try if it fixes for you, if not we just go with skipping it.